### PR TITLE
Argparser port

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: src/python/foglamp/core/scheduler.py [hunk 1]
```diff
@@ -627,6 +627,10 @@ def max_running_tasks(self, value: int)->None:
         # the start of the awaited coroutine.
         args = self._process_scripts[schedule.process_name]
 
+        # add core management port to process script args
```
Comment (ashwinscale) [inline src/python/foglamp/core/scheduler.py]: parhaps add port as well

File: src/python/foglamp/device/__main__.py [hunk 2]
```diff
@@ -16,25 +18,24 @@
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
-
+_logger = logger.setup("SouthBound Device", level=20)
```
Comment (ashwinscale) [inline src/python/foglamp/device/__main__.py]: Can we wait until the directory restructure for the southbound/northbound changes?

File: src/python/foglamp/device/__main__.py [hunk 3]
```diff
@@ -16,25 +18,24 @@
...
+parser.add_argument('--port',  required=True, help='Core Management Port')
+
+
+namespace = parser.parse_args(sys.argv[1:])
+
+plugin = '{0}'.format(namespace.name)
+
+core_management_port = int(namespace.port)
+
+print(plugin, core_management_port)
```
Comment (ashwinscale) [inline src/python/foglamp/device/__main__.py]: use logger instead of print

**Comment by praveen-garg** [comment]
@ashwinscale  Done with the feedback fixes

File: src/python/foglamp/core/scheduler.py (hunk 1 above)
Comment (praveen-garg) [reply src/python/foglamp/core/scheduler.py]: added

File: src/python/foglamp/device/__main__.py (hunk 2 above)
Comment (praveen-garg) [reply src/python/foglamp/device/__main__.py]: Keeping Device

File: src/python/foglamp/device/__main__.py (hunk 3 above)
Comment (praveen-garg) [reply src/python/foglamp/device/__main__.py]: added

**Comment by praveen-garg** [comment]
closing in the favour of https://github.com/foglamp/FogLAMP/pull/242

**Pull request closed without merging**